backend/core/src/secondary-adapters/postgres/video/video-repository-adapter.ts

```diff
@@ -292,4 +291,66 @@ export class VideoRepositoryAdapter implements VideoRepository {
       },
     });
   }
+
+  async getVideosBySearch({ searchText, duration, date, type, sortBy }: VideoSearch): Promise<DataVideo> {
+    const queryOrderByObject = {
+      orderBy: sortBy.map((arr) => ({
+        [arr[0]]: arr[1],
+      })),
+    };
```

Comment:
Try instead of this Prisma generated types.
it's like `Prisma.VideoOrderByWithRelationInput`.